executor/ddl.go

```diff
@@ -516,8 +521,52 @@ func (e *DDLExec) getRecoverTableByTableName(tableName *ast.TableName) (*model.J
 	return jobInfo, tableInfo, nil
 }
 
-func (e *DDLExec) executeFlashBackCluster(s *ast.FlashBackClusterStmt) error {
-	return dbterror.ErrGeneralUnsupportedDDL.GenWithStackByArgs("FLASHBACK CLUSTER")
+// ValidateFlashBackTS validates that readTS does not exceed the current time not strictly.
```

Comment:
Maybe `flashbackTS < currentTS && flashbackTS > gc_safe_ts` is a more accurate description.